src/Symfony/Component/JsonStreamer/Read/PhpGenerator.php

```diff
@@ -305,6 +305,16 @@ private function line(string $line, array $context): string
         return str_repeat('    ', $context['indentation_level']).$line."\n";
     }
 
+    /**
+     * @param string $identifier
+     *
+     * @return string
+     */
+    private function quote(string $identifier): string
```

Comment:
We don't add annotations if they don't add information that is not already in the method signature.
```suggestion
    private function quote(string $identifier): string
```